PatientProj/src/hospital/Gender.java

```diff
@@ -2,5 +2,7 @@
 
 public enum Gender {
 	MALE,
-	FEMALE;
```

Comment:
Why do we need this enum? Does temperature range vary based on male or female. Or does Allergy differ?